api/python/notebooks/analysis_demo/comp_bio_census_info.ipynb

```diff
@@ -18,6 +17,9 @@
     "- Census summary content tables\n",
     "- Understanding Census contents beyond the summary tables\n",
     "\n",
+    "⚠️ Note that the Census RNA data includes duplicate cells present across multiple datasets, these can be filtered in or out using the cell metadata variable `is_primary_data`.\n",
```

Comment:
run-on sentence.  Suggest breaking into two, e.g.,

> Note that the Census RNA data includes duplicate cells present across multiple datasets. Duplicate cells can be filtered in or out using the cell metadata variable `is_primary_data`.

I would also add a link to the normative section of the CxG schema that defines this attribute.

Reply:
+1